.github/workflows/ghci.yml

```diff
@@ -161,3 +184,10 @@ jobs:
 
             exit $exit_code
           fi
+
+
```

Comment:
There’s a double blank line here. Might be an accident.

Reply:
The Coverity step above already uses single blank lines to segment its shell code, so I used two blank lines to better set the steps apart. Maybe, we could move the Coverity logic into it's own script file and call that from the yml, but where should it be placed?